crates/nu-cli/src/completions/completion_common.rs

```diff
@@ -92,10 +96,21 @@ pub fn complete_item(
     partial: &str,
     cwd: &str,
     options: &CompletionOptions,
-) -> Vec<(nu_protocol::Span, String)> {
+    engine_state: &EngineState,
+    stack: &Stack,
+) -> Vec<(nu_protocol::Span, String, Option<Style>)> {
     let partial = surround_remove(partial);
     let isdir = partial.ends_with(is_separator);
     let cwd_pathbuf = Path::new(cwd).to_path_buf();
+    let ls_colors = (engine_state.config.completion_use_ls_colors
```

Comment:
For consistency's sake, what do you think about renaming this?
```suggestion
    let ls_colors = (engine_state.config.use_ls_colors_completions
```